spec/interactors/save_listing_spec.rb

```diff
@@ -2,14 +2,15 @@
 
 RSpec.describe SaveListing do
   let(:service_area)    { create :service_area }
+  let(:contact_method)  { create :contact_method_email }
   subject(:interaction) { SaveListing.run params }
 
   describe 'validation' do
     let(:params) {{
       type: '',  # only Listing field with validation
       service_area: service_area.id,
       person: {
-        preferred_contact_method: 'email',
+        preferred_contact_method: contact_method.id,
```

Comment:
why is this `contact_method.id` here, and not `contact_method`?

Reply:
The interactor allows for either an id or actual record (because its defined as a `record` type there), but the POST params will have the `id` so wanted to make that part of the test.

Reply:
ah! awesome.